### ISSUE TITLE
Comparison with null fails

### ISSUE THREAD

**Issue by skiptirengu** [lunet-io/scriban#189]
Applying this data:
```json
{
  "number_one": 1,
  "number_null": null
}
```

to the following template:
```jade
{{-
  if model.number_one != null
    "Number one is not null. "
  end
  if model.number_null == null
    "Number null is null"
  end
-}}
```

Produces wrong output:
- Expected: `Number one is not null. Number null is null`
- Actual: `Number null is null.`

Tested on the [online demo](https://scribanonline.azurewebsites.net/).

**Comment by ArtDubya** [comment]
Just wanted to add to this in case anyone else stumbles upon this issue... I had the same issue, but I resolved by just using `if object`
ex.
```
{{-
  if model.number_one
    "Number one is not null. "
  end
  if model.number_null == null
    "Number null is null"
  end
-}}
```
Will return the Expected output: `Number one is not null. Number null is null`

**Comment by xoofx** [comment]
Fixed in 2.1.3 coming later today.
